charts/shenyu/README.md

```diff
@@ -6,14 +6,151 @@ Helm deployment documentation written for Apache/Shenyu.
 
 [Apache/ShenYu](https://shenyu.apache.org/docs/index/) is an asynchronous, high-performance, cross-language, responsive API gateway.
 
-## Get Repo Info
+## Add Helm repository
 
 ```shell
 helm repo add shenyu https://apache.github.io/shenyu-helm-chart
 helm repo update
 ```
 
-_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+## Install
+
+### Deployment prerequisites
+
+Before reading this document, you need to read [Deployment prerequisites](https://shenyu.apache.org/docs/deployment/deployment-before/) to complete the environment preparation before deploying ShenYu.
+
+### Instructions
+
+* **Install the application**: By default, both admin and bootstrap are installed.
+* **Service Exposure**: Use NodePort to expose the service, the default port is `31095` for admin and `31195` for bootstrap.
+* **Database**: Currently supports h2, MySQL, PostgreSQL as database. Default is h2.
+
+### h2 as database
+
+Running the following command will install admin and bootstrap under shenyu namespace and create namespace.
+
+```shell
+helm install shenyu shenyu/shenyu -n=shenyu --create-namespace
+```
+
+### MySQL as database
+
+Modify and copy the following command and execute.
+
+```shell
+helm install shenyu shenyu/shenyu -n=shenyu --create-namespace \
+      --set dataSource.active=mysql \
+      --set dataSource.mysql.ip=127.0.0.1 \
+      --set dataSource.mysql.port=3306 \
+      --set dataSource.mysql.username=root \
+      --set dataSource.mysql.password=123456 
+```
+
+## PostgreSQL as database(Version of ShenYu > 2.5.0)
```

Comment:
may be `>=2.5.0` is better? what do you think of this?

Reply:
2.5.0(now) is not supported. There is something wrong with `2.5.0` of official image

Reply:
ok